addons/payment_adyen/static/src/js/payment_form.js

```diff
@@ -173,6 +173,35 @@ odoo.define('payment_adyen.payment_form', require => {
                             showPaymentMethods: true,
                             showPayButton: false,
                             setStatusAutomatically: true,
+                            onSelect: (component) => {
+                                if (component.props.name === "PayPal") {
+                                     if (!this.paypalForm){
+                                         // create div element to render PayPal component
+                                         const el = document.createElement("div");
+                                         document.getElementById(
+                                             `o_adyen_dropin_container_${paymentOptionId}`
+                                         ).appendChild(el);
+                                         el.setAttribute("id", "o_adyen_paypal_component");
+                                         el.setAttribute("className", "d-none");
+                                         // create and mount PayPal button in the component
+                                         const component = checkout.create("paypal").mount(
+                                             `#o_adyen_paypal_component`
+                                         );
+                                         this.paypalForm = this.$(`#o_adyen_paypal_component`);
+                                         this.txContext.tokenizationRequested = false;
+                                         component.acquirerId = paymentOptionId;
```

Comment:
`mount` returns the component, allowing you to chain as follows:
```suggestion
                                         checkout.create("paypal").mount(
                                             `#o_adyen_paypal_component`
                                         ).acquirerId = paymentOptionId;
                                         this.paypalForm = this.$(`#o_adyen_paypal_component`);
                                         this.txContext.tokenizationRequested = false;
```

Reply:
In this case, however, I don't believe you need to set the `acquirerId` at all.

Reply:
acquirerId is needed in dropinOnSubmit

Reply:
Because it's the component that is passed to `dropinOnSubmit`; ok!